tests/suites/ovs_maas/ovs_netplan_config.sh

```diff
@@ -7,7 +7,7 @@ run_ovs_netplan_config() {
 	# an address in 'space1'.
 	# - The third NIC is assigned an address in 'space2'.
 	juju switch controller
-	juju deploy cs:~juju-qa/bionic/space-invader --constraints='tags=ovs' --to lxd:0 --bind 'space1 invade-b=space2'
+	juju deploy cs:~juju-qa/bionic/space-invader --constraints='tags=ovs' --to lxd:0 --bind 'space1 invade-b=space2' --series jammy --force
```

Comment:
let's not use the old style URL with series in it
Use the charmhub version and focal so that --force is not required